pages.es/osx/ftxdiff.md

```diff
@@ -0,0 +1,16 @@
+# ftxdiff
+
+> Compara las diferencias entre dos fuentes.
+> Más información: <https://developer.apple.com/fonts>.
+
+- Envía las diferencias a un archivo de texto específico:
+
+`ftxdiff --output {{ruta/al/archivo_de_fontdif.txt}} {{ruta/al/archivo_ont_1.ttc}} {{ruta/al/archivo_font_2.ttc}}`
+
+- Incluir nombres de glifos en la salida:
+
+`ftxdiff --include-nombres-de-glifos`
```

Comment:
You accidentally translated options themselves lol.
```suggestion
`fftxdiff --include-glyph-names`
```

Reply:
Good catch